From d47870ed72960480744bec73de24c4d32593a43d Mon Sep 17 00:00:00 2001
From: Tom Pollard <tom.pollard@codethink.co.uk>
Date: Thu, 13 Aug 2026 12:36:07 +0100
Subject: [PATCH] crun.bst: Bump to 1.29.1 to integrate fix for #2050

This patch release includes a number of fixes, but importantly
includes:

https://github.com/containers/crun/commit/27fe1ced88c828060e3281a7
0bb29d332d17c3da

Which resolves the upstream issue reported for #2050:

https://github.com/containers/crun/issues/21823
---
 elements/components/crun.bst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/elements/components/crun.bst b/elements/components/crun.bst
index f8e4705c9..9f309f09b 100644
--- a/elements/components/crun.bst
+++ b/elements/components/crun.bst
@@ -26,7 +26,7 @@ sources:
 - kind: git_repo
   url: github:containers/crun.git
   track: '*.*'
-  ref: 1.29-0-gd66af6a0fc2ab5fabd951b427a292344632ce074
+  ref: 1.29.1-0-gf0d911de5587342cfeb16473bf32ecdfeaf25957
 - kind: git_module
   path: libocispec
   url: github:containers/libocispec.git
-- 
2.55.0

